### ISSUE TITLE
Stop not clearing playback session in dashboard if playback was already paused

### ISSUE THREAD

**Issue by ilia-21** [jmshrv/finamp#961]
Hello! Thank you for making this amazing app!
I'm using it with [jellyfin-rpc](https://github.com/Radiicall/jellyfin-rpc/) to show my activity on discord. I noticed that when I pause the song and stop playback it still shows up in my profile. jellyfin-rpc takes data from the dashboard so I checked it, and there still was my session. I tried closing the app, even killing it, session was still there. Important that it doesn't happen if I stop the playback (swipe down) while song is still playing.

Repro steps:
1. Go to settings > Audio service > Report current queue to server?: ON
2. Play a song
3. Pause it
4. Swipe down to stop playback
5. Open jellyfin dashboard to see that the song is still reported

**Comment by Chaphasilor** [comment]
For me on Android stopping actually never clears the session. Stopping without pausing resets the position to 0 seconds, and stopping after pausing first won't change the position at all.  
In neither caseis the session cleared, which is definitely a bug.

Which platform are you on?

**Comment by ilia-21** [comment]
Hello, thank you for reply.
Im on Android 14, (Galaxy A15 with One UI 6.1 if this is relevant)

Looks like you are right, after I stop playback if the song is playing it will show this:
![image](https://github.com/user-attachments/assets/3fd9a424-3512-450e-a3ef-7e0df53cf368)
Then after a a minute or so as you said it will report as paused on 0 seconds.
I tried changing Audio Service > Playback Session Update Frequency to a smaller number, and looks like it is the exact time between no playback and reporting as paused on 0 seconds. So for example if I set Update Frequency to 100 seconds: 
1. I stop playback
2. Shows as session ended
3. 100 seconds later it shows that playback is paused at 0 seconds

Also another thing that maybe important is if I close the app right after stopping playback it won't send my session to the server

**Comment by Chaphasilor** [comment]
Interesting, in my case the dashboard never shows the playback as stopped. It just shows as paused and then after "update frequency" seconds resets to 0 seconds.

Getting the update to properly reflect the status should be easy, I'm not sure about the rest of the behavior. I'll take a lool tomorrow, since I want playback reporting to be as solid as possible!

**Comment by HamletDuFromage** [comment]
I'm unsure if it's the same issue, if it isn't I'll open a new one. But I paused this song, put Finamp in the background and after 30+ minutes it's still showing in the activity tab. Finamp settings mention that after 5 minutes, jellyfin stops reporting the client but it's obviously not happening. I'm unsure if it's a jellyfin or Finamp issue.
![image](https://github.com/user-attachments/assets/7fb0c503-9df1-4c4b-922e-f5c771a978f6)

**Comment by Chaphasilor** [comment]
@HamletDuFromage that is actually by design! As long as Finamp is running and "playing" something (not stopped), it will also keep the session in the server dashboard active.

**Comment by HamletDuFromage** [comment]
Ah I see, my bad then :o
Can playback be stopped from the app itself? I guess I always just use the pause button whenever I stop listening to music.

**Comment by Chaphasilor** [comment]
No worries!  
You can stop by swiping down on the "mini player" bar at the bottom. It might trigger your phone's one-handed mode, but should stop playback (the bar will disappear).  
We are aware that it's not ideal and will add a proper button for this in the future :)

**Comment by HamletDuFromage** [comment]
Playback did indeed return to 0 after swiping down, and in term I assume will disappear from the dashboard. Thanks :)
I guess I would like a timeout feature, where the client closes the session after n minutes/hours of inactivity but maybe it's just personal nitpick. It certainly doesn't matter much. Awesome piece of software in any case, thank you for your contributions to it.

**Comment by ilia-21** [comment]
\>We are aware that it's not ideal and will add a proper button for this in the future :)

Off topic, but it felt natural for me when I first used the app. I tried to stop playback by swiping left/right, but it changed song instead, so swiping down was the only option I could think of and it worked

**Comment by Chaphasilor** [comment]
@ilia-21 yeah we'll definitely keep that gesture, but it certainly isn't super obvious to find. A properly labeled button in addition to the gesture should fix that.
